icepyx/__init__.py

```diff
@@ -1 +1 @@
-from .core import query
+from icepyx.core.query import Query
```

Comment:
Right, as mentioned at https://github.com/icesat2py/icepyx/pull/116#discussion_r470849508_: Could you please change the following files to use the new 'import icepyx as ipx' syntax:

- icepyx/tests/behind_NSIDC_API_login.py
- icepyx/tests/is2class_query.py
- icepyx/tests/test_granules.py

P.S. It might be easier to use a Git GUI to do difficult git tasks like rebase, cherry-pick and so on. Personally I use [Gitkraken](https://support.gitkraken.com/#gitkraken-git-gui-tutorial), and it really makes tracking changes a lot easier than on the command line.

Reply:
I saw your comment and was working on making the changes as you re-posted it here. Thanks!

Reply:
Cool, I found a few more instances of 'from icepyx import icesat2data as ipd':

- doc/source/dev-notebooks/ICESat-2_DAAC_DataAccess_working.ipynb
- doc/source/dev-notebooks/is2_demo_download_restart.ipynb
- doc/source/dev-notebooks/spatial_subsetting_vis.ipynb

Reply:
Also these lines here:

https://github.com/icesat2py/icepyx/blob/d490ab5cd9de736949b80827becf659c8d642f93/icepyx/core/geospatial.py#L27-L31

Reply:
Great job catching all of these - don't know where my brain was on Friday!

I did not bother to update the doc/source/dev-notebooks ones because they're predominantly scratch spaces we were using to share and test new functionality among developers, so there are lots of things in them that don't work and I don't bother with keeping them "updated" if I'm not using them.